opengrok-tools/src/main/python/opengrok_tools/mirror.py

```diff
@@ -177,11 +182,13 @@ def main():
                 except KeyboardInterrupt:
                     sys.exit(1)
                 else:
-                    if any([True for x in project_results if x == 1]):
-                        ret = 1
+                    if any([True for x in project_results if x == FAILURE_EXITVAL]):
+                        ret = FAILURE_EXITVAL
+                    if all([True for x in project_results if x == CONTINUE_EXITVAL]):
```

Comment:
I think you need to check indeed for all. Something like:
```python
(len([x for x in project_results if x == 2]) == len(project_results))
```

Reply:
or something like:
```python
 all(list(map(lambda x: True if x == 1 else False, project_results)))
```

Reply:
yes I discovered the problem also

Reply:
I went for `all([x == CONTINUE_EXITVAL for x in project_results])`

Reply:
which is essentially your last suggestion